js/scenes/Item.js

```diff
@@ -56,7 +57,9 @@ class Item extends Component {
         style={styles.image}
         source={{ uri: image }}
         resizeMode="cover"
-        ></Image>
+        >
+        <SocialShare/>
```

Comment:
Why are no props passed. IIUC this usage will result in the component hitting the `Object.assign({}, this.props);` line and thus no props will be passed to the `.share()` method?

Perhaps I've missing something, but I'd prefer to see something more explicit like:

```js
<SocialShare title={`Sharing {this.props.itemId}` from Magnet }/>
```

It's clear to me when reading what the expected outcome will be.